mma-ctf-2nd-2016/forensic/broken-ntfs-500/README.md

```diff
@@ -0,0 +1,38 @@
+# MMA CTF 2nd 2016 : broken-ntfs-500
+
+**Category:** Forensic
+**Points:** 500
+**Solves:** 3
+**Description:**
+
+> Someone broke my disk!!
+> 
+> I discovered this command from .bash_history.
+> 
+> 
+> openssl aes-256-cbc -e -in /tmp/flag.jpg -out /mnt/flag:flag -pass file:<(openssl aes-256-cbc -e -in ./key -pass pass:`pwd`/key -nosalt)
+> 
+> Please recover flag from [[problem.7z](./problem.7z)]([problem.7z](./problem.7z)).
+> 
```

Comment:
problem.7z looks like a bigger file so it has been automatically added to the gitignore, so add it somewhere else in the cloud and put the link here